docs/public/deploying-java-app-with-database.adoc

```diff
@@ -0,0 +1,432 @@
++
+[source,sh]
+----
+$ minikube start --force --driver=docker --kubernetes-version=v1.19.8
+----
+
+* If you are a non-root user, start minikube as usual:
++
+[source,sh]
+----
+$ minikube start --kubernetes-version=v1.19.8
+----
+
+==== Configuring minikube
+The application requires an ingress add-on to allow the routes to be created easily. Configure minikube for ingress by adding ingress as a minikube add-on:
+[source,sh]
+----
+> minikube addons enable ingress
+----
+
+It is possible that you face the Dockerhub pull rate limit if you do not have a pull secret for your personal free Dockerhub account. During ingress initialization two of the job pods used by ingress may fail to initialize due to pull rate limits. If this happens, and ingress fails to enable, you add a secret for the pulls for the following service accounts:
+
+* ingress-nginx-admission
+* ingress-nginx
+
+to add a pull secret for these service accounts: 
+
+. Switch to the kube-system context:
++
+[source,sh]
+----
+$ kubectl config set-context --current --namespace=kube-system
+----
+
+. Create a pull secret:
++
+[source,sh]
+----
+$ kubectl create secret docker-registry regcred --docker-server=<your-registry-server> --docker-username=<your-name> --docker-password=<your-pword> --docker-email=<your-email>
+----
++
+where:
+* <your-registry-server> is the DockerHub Registry FQDN. (https://index.docker.io/v1/)
+* your-name> is your Docker username.
+* <your-pword> is your Docker password.
+* <your-email> is your Docker email.
+
+. Add this new cred (`regcred` in the example above) to the default service account in minikube:
++
+[source,sh]
+----
+$ kubectl patch serviceaccount ingress-nginx-admission -p '{"imagePullSecrets": [{"name": "regcred"}]}'
+$ kubectl patch serviceaccount ingress-nginx -p '{"imagePullSecrets": [{"name": "regcred"}]}'
+----
+
+ Default Service Account Pull Secret patch:
+ Much like the ingress service accounts, the default service account will need to be patched with a pull secret configured for your personal docker account. 
+
+* switch to the default context:
++
+[source,sh]
+----
+$ kubectl config set-context --current --namespace=default
+----
+
+* create the same docker-registry secret configured for your docker , now for the default minikube context:
++
+[source,sh]
+----
+$ kubectl create secret docker-registry regcred --docker-server=<your-registry-server> --docker-username=<your-name> --docker-password=<your-pword> --docker-email=<your-email>
+----
++
+where:
++
+** <your-registry-server> is the DockerHub Registry FQDN. (https://index.docker.io/v1/)
+** <your-name> is your Docker username.
+** <your-pword> is your Docker password.
+** <your-email> is your Docker email.
+
+* Add this new cred ('regcred' in the example above) to the default service account in minikube:
++
+[source,sh]
+----
+$ kubectl patch serviceaccount default -p '{"imagePullSecrets": [{"name": "regcred"}]}'
+----
+
+Kubernetes Dashboard graphical UI config:<br>
```

Comment:
This `<br>` shows as-is even in the rich diff view of PR. I hope it won't show up in the docs.

Reply:
This is still an issue.